src/ticket/lib/ticketReducer.ts

```
@@ -0,0 +1,78 @@
+          ? {
+              ...ticket,
+              comments: [
+                ...ticket.comments,
+                {
+                  id,
+                  content: action.payload.comment,
+                },
+              ],
+            }
+          : ticket,
+      );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)